srcpkgs/brother-mfcj6720dw-cupswrapper/INSTALL

```diff
@@ -0,0 +1,8 @@
+case "${ACTION}" in
+post)
+    mkdir -p /usr/share/ppd/cupsfilters
```

Comment:
I had to add this line to the INSTALL I copied from `brother-dcp197c-cupswrapper`, maybe it should be added back in that package ? The brother install script expectd that ppd dir to exist. On the other hand, it's created by the package `cups-filters` so maybe this package should just be added as a dependency, both of my package and of `brother-dcp197c-cupswrapper`.